prod/westeurope/internal/api/storage_app/queue_profilemigrateservicepreferences/terragrunt.hcl

```diff
@@ -0,0 +1,17 @@
+dependency "storage_account" {
+  config_path = "../account"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_storage_queue?ref=v3.0.3"
+}
+
+inputs = {
+  name                  = "profile-migrate-service-preferences-from-legacy"
```

Comment:
`service-preferences` or `services-preferences`?

Reply:
I'll let @balanza decide

Reply:
Switched to plural in [0a05c54](https://github.com/pagopa/io-infrastructure-live-new/pull/578/commits/0a05c54d5db0f19fcb6850f95c1e3f67d4988ed2)